pyne/tally.pyx

```diff
@@ -385,15 +448,20 @@ cdef class Tally:
             (<cpp_tally.Tally *> self._inst).normalization = <double> value
 
 
-    property particle_name:
-        """no docstring for particle_name, please file a bug report!"""
+    property particle_names:
+        """no docstring for particle_names, please file a bug report!"""
```

Comment:
Why don't you write a docstring for this?

Reply:
this is how it was before, I don't know why there is not doctoring there on this one or the others....

Reply:
I see that they are all that way... I guess I feel like it makes sense to correct it when the opportunity presents itself - and this might be such an opportunity

Reply:
Should I wait for a docstring?

Reply:
yes please !